pkg/network/master/subnets.go

```diff
@@ -87,6 +140,13 @@ func (master *OsdnMaster) addNode(nodeName string, nodeIP string, hsAnnotations
 	}
 
 	// Create new subnet
+	annotations := make(map[string]string)
+	for key, value := range hsAnnotations {
+		annotations[key] = value
+	}
```

Comment:
Same here, you don't have to copy existing annotations in hsAnnotations. 
Just add the new annotation to hsAnnotations (similar to subnet.Annoations)